binding.gyp

```diff
@@ -49,18 +62,18 @@
         "src/node_sqlite3.cc",
         "src/statement.cc"
       ],
-      "defines": [ "NAPI_VERSION=<(napi_build_version)", "NAPI_DISABLE_CPP_EXCEPTIONS=1" ]
+      "defines": [ "NAPI_DISABLE_CPP_EXCEPTIONS=1", "NODE_API_SWALLOW_UNTHROWABLE_EXCEPTIONS" ]
```

Comment:
@deepak1556 I wonder if `NAPI_VERSION=<(napi_build_version)` is needed, I noticed we do not set it in our fork

Reply:
The value is only needed by `node-pre-gyp` https://github.com/mapbox/node-pre-gyp/blob/master/README.md#the-napi_build_version-value which we don't use, so I removed it.